pkg/plugins/manager/signature/manifest.go

```diff
@@ -341,6 +342,9 @@ func (s *Signature) Verify(ctx context.Context, keyID string, block *clearsign.B
 	if _, err = openpgp.CheckDetachedSignature(keyring,
 		bytes.NewBuffer(block.Bytes),
 		block.ArmoredSignature.Body, &packet.Config{}); err != nil {
+		if errors.Is(err, openpgpErrors.ErrUnknownIssuer) && len(keyring) > 0 && len(keyring[0].Revocations) > 0 {
+			return fmt.Errorf("%v (KeyID: %s): %w", openpgpErrors.ErrKeyRevoked, keyID, err)
```

Comment:
I think you want
```suggestion
			return fmt.Errorf("%w (KeyID: %s): %w", openpgpErrors.ErrKeyRevoked, keyID, err)
```

Reply:
I wanted to wrap the original error and not the new one I am generating (in any case, it doesn't work properly with `errors.Is`, see my other comment)

Reply:
Ah I see what you mean

Nit: To make it less confusing, I would change the format string and value to make it more explicit that you just want the error string from ErrKeyRevoked rather than the `error` itself:

```go
return fmt.Errorf("%s (KeyID: %s): %w", openpgpErrors.ErrKeyRevoked.Error(), keyID, err)
```